docs/Operators.md

```diff
@@ -7135,6 +7136,153 @@ Other versions of this operator: <a href="Changelog.md#If-1">If-1</a>
 </dl>
 
 
+### <a name="ImageToCol"></a><a name="imagetocol">**ImageToCol**</a>
```

Comment:
Pytorch name "unfold" looks to be better than this one, since it does not have to be an image, and it's indeed unfolding a window/block.

Reply:
Yes, I agree. I will update the operator name.